.bash_profile

```diff
@@ -0,0 +1,22 @@
+# Custom bash prompt via kirsle.net/wizards/ps1.html
+export PS1="\[$(tput bold)\]\[$(tput setaf 4)\][\[$(tput setaf 5)\]\u\[$(tput setaf 4)\]@\[$(tput setaf 5)\]\h \[$(tput setaf 2)\]\W\[$(tput setaf 4)\]]\\$ \[$(tput sgr0)\]"
+
+# To make nvim executable
+ PATH="/Users/ingridsuria/tools/neovim/0.4.3/bin:$PATH"
```

Comment:
Change "/Users/ingridsuria" to "~" or "$HOME" and check that it still works. Otherwise it only works on that particular machine, where your username is ingridsuria and probably only work on MacOS.